build-container-image/action.yaml

```
@@ -53,6 +54,12 @@ runs:
       run: |
         set -euo pipefail
 
+        # Validate that the build arguments input doesn't contain any whitespaces
+        if [[ $BUILD_ARGUMENTS = *" "* ]]; then
```

This could be simplified with grep (IMO, grep is easier to read than the special bashism):

```suggestion
        # Validate that the build arguments input doesn't contain any whitespaces
        if echo "$BUILD_ARGUMENTS" | grep '[[:space:]]'; then
```

> [!TIP]
> Also works with tabs. Tested with:
> ```sh
> VAR1=$(echo -e "text\twith\tspace")
> VAR2="textwithoutspace"
> if echo "$VAR1" | grep '[[:space:]]'; then echo nonononono; fi
> if echo "$VAR2" | grep '[[:space:]]'; then echo nonononono; fi
> ```

---

Updated in f7ce890.